src/gfx/meshes/MeshCreator.js

```diff
@@ -14,7 +14,7 @@ function _createInstancedCylinders(useZSprites, openEnded, useTraceMesh) {
     Geometry(a, b) {
       return new geometries.Instanced2CCylindersGeometry(a, b, useZSprites, openEnded);
     },
-    Object: useTraceMesh ? meshes.Trace : meshes.ZSprite,
+    Object: useTraceMesh ? meshes.Instanced : meshes.ZSprite,
```

Comment:
Thank you for clarifying our meshes types!
I've got just one question about the actuality of parameter `useTraceMesh ` in the function `_createInstancedCylinders`.  Could we use `useZSprites` parameter for controlling the choice between instanced cylinders and instanced+sprite cylinders? Maybe we can generalize the logic here? It's would be great if we avoid specific parameters as `trace` here. Any ideas?